frontend/components/CourseFieldSelector.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="courses">
+    <select class="selector" v-model="myChoice" name="Kierunek">
+      <option :value="null" disabled hidden>Kierunek</option>
+      <option value="Informatyka">Informatyka</option>
+      <option value="Grafika">Grafika</option>
+      <option value="Ekonomia">Ekonomia</option>
+    </select>
+  </div>
+</template>
+
+<script>
+export default {
+  el: "#courses",
+  data() {
+    return {
+      myChoice: null,
```

Comment:
Please, use more descriptive name. I mentioned some examples before.

Reply:
Changed to selectedCourse.